### PR TITLE
Stitches doc

### PR DESCRIPTION
**Pull request by clairehhelms**
short documentation about using some stitches functionality for ui kit users. Please check for info that I may have missed or is confusing!

### CODE REVIEW

**Review by artmsilva — APPROVED** [review]

**Comment by clairehhelms** [comment]
// go to resources -> using stitches to style your products :-)

File: docs/resources/architecture/themes.mdx [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+
+// Use it
+<div className={newButton()}>I'm new!</div>;
+```
+
+\*\*note: style property names use camel case in these functions (example: `z-index` becomes `zIndex`)
+
+## Keyframes
+
+Another function available in Stitches is the `keyframes` function. Using the keyframe animation technique, Stitches allows the creation of global CSS keyframe rules, which let transitions be easily reused across code. Below is an example of a simple fading-out animation:
```
Comment (atomictangerine) [inline docs/resources/architecture/themes.mdx]: nit: which `lets` transitions...

File: docs/resources/architecture/themes.mdx [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+	marginTop: theme.space["200"],
+	// use the color token $primary, which is equivalent to rgba(17, 17, 17, 1)
+	// color tokens change automatically when a color theme is applied
+	color: theme.colors.primary,
+});
+```
+
+## Breakpoints
+
+Stitches is also capable of defining breakpoints, which are reusable media queries useful for responsive styling. The WPDS UI Kit supports a variety of size breakpoints and custom themes, which can be explored [here](https://github.com/washingtonpost/wpds-ui-kit/blob/main/ui/theme/src/stitches.config.ts). The button in the example below would be uniquely styled when the viewport is smaller than `768px`, or `sm`.
```
Comment (atomictangerine) [inline docs/resources/architecture/themes.mdx]: Personally, I'm not a big fan of having the word `here` be a link out. Perhaps you could say 'which can be explored in the [stitches.config.ts](https://github.com/washingtonpost/wpds-ui-kit/blob/main/ui/theme/src/stitches.config.ts)' or something like that

**Review by atomictangerine — CHANGES_REQUESTED** [review]
Looks good! I would suggest not using words like `here` to link out to other pages, but instead use something more descriptive that lets people know where they're going.
That being said, I LOVE THE EXAMPLES. Great job. Super easy to follow and so much easier to understand that the actual stitches docs.

**Review by artmsilva — APPROVED** [review]

**Comment by clairehhelms** [comment]
Will merge upon updating with styling based on parent element tags (eg for dark mode)

**Review by atomictangerine — APPROVED** [review]

**Pull request merged**